app/components/primer/auto_complete.rb

```diff
@@ -2,6 +2,8 @@
 
 module Primer
   # Use `AutoComplete` to populate input values from server search results.
+  # @accessibility
+  #   Always provide a label for the `Autocomplete` component.
```

Comment:
Is this something that should be built into the component, so users can't make the component inaccessible?

Reply:
Yep! I plan to follow up in a separate PR for this component.